### PR TITLE
v2.0.2 (CHANGELOG + version bump)

### PR DESCRIPTION
**Pull request by intellectronica**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -13,4 +21,4 @@ Development
 2.0 (2015-10-13)
 ----------------
 * Brand new client, entirely rewritten from scratch and now completely dockerized
-* Ready for the new Aldryn baseproject (v3)
\ No newline at end of file
+* Ready for the new Aldryn baseproject (v3)
```
Comment (intellectronica) [inline CHANGELOG.rst]: I don't understand why the diff here. The two lines are really identical.

Comment (Chive) [inline CHANGELOG.rst]: probably there was no empty line at the end of the file

**Comment by Chive** [comment]
Thanks! I'll quickly fix the typo and then merge it :)

File: CHANGELOG.rst (hunk 1 above)
Comment (intellectronica) [inline CHANGELOG.rst]: Ah yes, my Emacs adds those automatically.

**Comment by intellectronica** [comment]
Which typo @Chive ?

**Comment by Chive** [comment]
@intellectronica https://github.com/aldryn/aldryn-client/commit/96a3f830c4854d446169fc2faa6643c1258827c3 ``agagin``

File: CHANGELOG.rst (hunk 1 above)
Comment (Chive) [inline CHANGELOG.rst]: :+1:

**Pull request merged**